src/github.com/getlantern/golog/golog.go

```diff
@@ -55,9 +59,9 @@ type Logger interface {
 	Debugf(message string, args ...interface{})
 
 	// Error logs to stderr
-	Error(arg interface{})
+	Error(arg interface{}) error
```

Comment:
Small convenience change to interface.  This allows us to log and return the error in one step.  The Errorf version of this replaces:

```golang
err := fmt.Errorf("Oh oh %v", arg)
log.Error(err)
return nil, err
```

with

```golang
return nil, log.Errorf("Oh oh %v", arg)
```